Reference-Platform/Platforms/Enterprise/ReleaseNotes-17.12.md

```diff
@@ -15,20 +15,18 @@ The goal of the Linaro Enterprise Reference Platform is to provide a fully teste
 
 ## Debian
 
-- Network Installer based on Debian 8.9 "Jessie"
-- Unified Reference Platform Kernel based on 4.12
+- Network Installer based on Debian 9.2 "Stretch"
+- Unified Reference Platform Kernel based on 4.14
 
 ## Enterprise Components
 - Docker 1.12.6
-- ERP 17.12 OpenStack Newton
-  - New Swift package released
-- Ceph 10.2.7
+- Ceph 12.2
 - Bigtop 1.2 stack (Hadoop 2.7.3, Spark 2.1, Hive 1.1.3)
 - ELK 5.4.1
 - OpenJDK 8
-- Libvirt 3.4.0
-- QEMU 2.9
-- DPDK 17.05
+- Libvirt 3.8.0
+- QEMU 2.10
+- DPDK 17.08
```

Comment:
We are not sure about this version, but let's go with 17.08 as tentative and amend later if we can include 17.11 instead.